app/views/SettingsView/index.js

```diff
@@ -221,10 +226,15 @@ class SettingsView extends React.Component {
 						right={this.renderDisclosure}
 					/>
 					<Separator />
-					<ListItem title={I18n.t('Version_no', { version: getReadableVersion })} testID='settings-view-version' />
+					<ListItem
+						title={I18n.t('Version_no', { version: getReadableVersion })}
+						onPress={() => this.saveToClipboard(getReadableVersion)}
+						testID='settings-view-version'
+					/>
 					<Separator />
 					<ListItem
 						title={I18n.t('Server_version', { version: server.version })}
+						onPress={() => this.saveToClipboard(server.version)}
```

Comment:
I think we should try to prevent that anonymous function, like this:
```jsx
onPress={this.copyServerVersion}
```

And then on the new `copyServer` function, you get the content you want like:
```jsx
const { server } = this.props;
this.saveToClipboard(server.version)
```

And the same for the app version.

Reply:
Thanks for this review. I made the changes. Sir, so should we try to avoid using anonymous functions whenever possible?